### PR TITLE
Added/improved obfuscation of some chunk handling classes

### PR DESCRIPTION
**Pull request by Siphalor**

### CODE REVIEW

File: mappings/net/minecraft/world/chunk/ChunkPos.mapping [hunk 1]
```diff
@@ -19,3 +19,5 @@ CLASS bby net/minecraft/world/chunk/ChunkPos
 		ARG 1 o
 	METHOD f getXEnd ()I
 	METHOD g getZEnd ()I
+	METHOD h getRegionX ()I
+	METHOD i getRegionY ()I
```
Comment (asiekierka) [inline mappings/net/minecraft/world/chunk/ChunkPos.mapping]: ... Y?

File: mappings/net/minecraft/world/chunk/storage/RegionFileCacheVersioned.mapping [hunk 2]
```diff
@@ -0,0 +1,10 @@
+CLASS bsx net/minecraft/world/chunk/storage/RegionFileCacheVersioned
```
Comment (asiekierka) [inline mappings/net/minecraft/world/chunk/storage/RegionFileCacheVersioned.mapping]: That's the wrong prefixing order

**Review by asiekierka — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/world/chunk/ChunkPos.mapping (hunk 1 above)
Comment (Siphalor) [reply mappings/net/minecraft/world/chunk/ChunkPos.mapping]: Whoops XD

File: mappings/net/minecraft/world/chunk/storage/RegionFileCache.mapping [hunk 3]
```diff
@@ -1,2 +1,17 @@
 CLASS bta net/minecraft/world/chunk/storage/RegionFileCache
+	FIELD d regionFiles Lit/unimi/dsi/fastutil/longs/Long2ObjectLinkedOpenHashMap;
 	METHOD a getRegionFile (Lbby;)Lbsz;
+		ARG 2 regionId
+		ARG 4 regionFile
+		ARG 5 regionFolder
+		ARG 6 mcaFile
+		ARG 7 regionFile2
```
Comment (Gegy) [inline mappings/net/minecraft/world/chunk/storage/RegionFileCache.mapping]: What is this parameter?

Comment (Siphalor) [reply mappings/net/minecraft/world/chunk/storage/RegionFileCache.mapping]: This is a local variable for a new RegionFile if the first one doesn't exist.
That's due to Mojang making everything final.

**Comment by Siphalor** [comment]
I'm actually not quite sure wether the ChunkSaveHandlerImpl should be renamed.
I think it's a bit misleading because it also handles loading of chunks but I don't know a better name for it.

**Comment by asiekierka** [comment]
Unfortunately, a different PR conflicted with most of this. I tried to re-integrate the interesting things, though.

**Pull request closed without merging**